packages/fast-components-react-base/src/toggle/toggle.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, {HandledProps} from "../foundation";
+
+import {IToggleProps} from "./toggle.props";
+import {IToggleClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+/**
+ * Toggle state interface
+ */
+export interface IToggleState {
+    checked: boolean;
+}
+/**
+ * Toggle base component
+ */
+/* tslint:disable-next-line */
+class Toggle extends Foundation<IToggleProps & IManagedClasses<IToggleClassNameContract>,  React.AllHTMLAttributes<HTMLElement>, IToggleState> {
```

Comment:
We should try to be as specific as possible here. The reason that button supported All was due to the fact that there were serious conflicts when trying to allow HTMLAnchor | HTMLButton. Here though, I don't we lose the purpose of this if we tell the user they can pass an href.